website/static/service-worker.js

```diff
@@ -9,8 +9,11 @@ importScripts(
 
 const plugins = [
   ...new Set(Object.values(parsersLocation).map((file) => `lib/${file}`)),
-  "lib/plugins/estree.js",
 ];
+// TODO: Remove after release v3
```

Comment:
@fisker Do you know the other workaround?

Reply:
Maybe we can use `prettier.version`.